packages/assets-controllers/src/token-prices-service/codefi-v2.ts

```diff
@@ -342,7 +342,8 @@ export class CodefiTokenPricesServiceV2
           ];
 
         if (!price) {
-          throw new Error(
+          // console error instead of throwing to not interrupt the fetching of other tokens in case just one fails
+          console.error(
             `Could not find price for "${tokenAddress}" in "${currency}"`,
```

Comment:
This used to throw an error; 
Noticed that on the import flow, if you try to import two tokens exp (USDC and STEELO) where it should succeed to fetch prices for USDC and not for STEELO. It will throw for STEELO preventing fetching for USDC; hence you end up on the home page where you do not see the user token amount for USDC.

Steps to REPRODUCE:
1- Go to home page on extension (IDEALLY remove the extension and import again so u don't have cached rate values)
2- search for STEELO and select it
3- search for USDC and select it
4- Click next and click import
5- go to home page and notice you do not see the USDC token amount in fiat


cc @mcmire

Reply:
Hmm, good catch. Would you mind putting this in another PR? This way it's more visible and we can track this as a separate changelog item.

Reply:
Thanks @mcmire  here is the link to the other PR https://github.com/MetaMask/core/pull/3687

Reply:
I will remove this change from this PR, then rebase once https://github.com/MetaMask/core/pull/3687 is merged 🙏